src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1713,9 +1713,9 @@ internal enum ErrorCode
         ERR_LanguageVersionDoesNotSupportDefaultInterfaceImplementationForMember = 8706,
 
         ERR_RuntimeDoesNotSupportProtectedAccessForInterfaceMember = 8707,
-        ERR_NotBaseOrImplementedInterface = 8708,
-        ERR_NotImplementedInBase = 8709,
-        ERR_NotDeclaredInBase = 8710,
+        //ERR_NotBaseOrImplementedInterface = 8708,
+        //ERR_NotImplementedInBase = 8709,
+        //ERR_NotDeclaredInBase = 8710,
 
```

Comment:
I think it may be better to just delete these, since it will likely be inappropriate to include these diagnostics in the C# 8.0 range of codes if it is introduced in a later language version.

Reply:
One possible reason to keep these error codes is to make it easier to revert this PR later on (to re-introduce the feature). We can reserve those error codes.

---
In reply to: [280623450](https://github.com/dotnet/roslyn/pull/35456#discussion_r280623450) [](ancestors = 280623450)